run-upgrade-tests.sh

```diff
@@ -70,7 +70,7 @@ pifpaf_stop(){
 
 cleanup(){
     pifpaf_stop
-    rm -rf $GNOCCHI_DATA
+    rm -rf -v $GNOCCHI_DATA
```

Comment:
unrelated :)

Reply:
yea, it was added for some debugging purpose and I forgot to remove it, thanks.